WordPress/gutenberg
# [RNMobile] Add iOS platform version to iOS local e2e capabilities

## Description


This PR adds the `platformVersion` capability explicitly to the desired capabilities list when running the e2e tests locally. Although this capability is not mandatory when using an iOS simulator, we're going to add it in case it helps to reduce the flakiness when running the e2e tests, as we have experienced recently.

For this purpose, I added an extra step when setting up the driver for running the tests. Now it fetches the latest iOS platform version and uses it as the `platformVersion` capability. 

Additionally when running the tests, it also prints the available iOS platform versions, here as an example:
```
...
Available iOS platform versions: [ 'iOS 14.2', 'iOS 14.0', 'iOS 13.7' ]
Using iOS 14.2 platform version
...
```

## How has this been tested?



For testing if the setup picks the latest iOS platform, it's recommended to install additional iOS simulator versions. This can be done from the Xcode app in `Xcode/Preferences/Components/Simulators`.

1. Run `npm run native test:e2e:ios:local`.
2. Observe that the command prints the available iOS platform versions and that picks the latest one.
3. Observe that all the e2e tests pass.

## Screenshots 
N/A

## Types of changes




Bug fix

## Checklist:
